learning/tour-of-beam/frontend/lib/pages/tour/widgets/unit_content.dart

```diff
@@ -177,8 +178,11 @@ class _ChallengeButtons extends StatelessWidget {
               hints: hints,
             ),
           ),
-        // TODO(nausharipov): solution button
-        Container(),
+        if (solution != null)
```

Comment:
If some logic can be put into `TourNotifier`, it's better to put it there. Something like `if (tourNotifier.hasSolution) {`